### PR TITLE
Addition of vxlan tests

### PR DESCRIPTION
**Pull request by Vibaswan**
### Tests Added:
1.  python test
2.  go test

### CODE REVIEW

File: features/flows/headers/vxlan/test_vxlan_header_with_inner_ipv6_traffic.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        captured_packets.validate_field(
+            "ethernet type", i, 62, api.num_to_bytes(34525, 2)
+        )
+
+        # inner ipv6 header
+        captured_packets.validate_field(
+            "ipv6 payload length",
+            i,
+            68,
+            api.num_to_bytes(tc["pktSize"] - 14 - 4 - 20 - 8 - 8 - 14 - 40, 2),
```
Comment (Vibaswan) [inline features/flows/headers/vxlan/test_vxlan_header_with_inner_ipv6_traffic.py:171]: ## Observation 
I saw that the payload length of this particular packet is coming out to be 148

Now I assume it should come out to be 144 

Calculation to support that the above statement
eth(14) + eth-payload(4) + ip(20) + udp(8) + vxlan(8) + inner-eth(14) + inner-eth-payload(4) + ipv6-header(40) = 112
Total packet(size) 126 - 112 = 144 (I see this value in Ixn Packet editor as well)

For now i was not sure for the calculation so pushed the changes with 148 (if theres is a bug I guess the inner eths payload is not being taken into the account maybe ?)

Comment (ashutshkumr) [reply features/flows/headers/vxlan/test_vxlan_header_with_inner_ipv6_traffic.py:171]: I just noticed that in test you're using:
```
api.num_to_bytes(tc["pktSize"] - 14 - 4 - 20 - 8 - 8 - 14 - 40, 2)
```

you should instead use:
```
api.num_to_bytes(tc["pktSize"] - 14 - 20 - 8 - 8 - 14 - 40, 4)
```

Comment (Vibaswan) [reply features/flows/headers/vxlan/test_vxlan_header_with_inner_ipv6_traffic.py]: issue opened 
https://github.com/open-traffic-generator/ixia-c/issues/112

File: features/flows/headers/vxlan/test_vxlan_header_with_inner_ipv6_traffic.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        f1.packet.ethernet().ipv4().udp().vxlan().ethernet().ipv6().tcp()
+    )
+
+    eth1.src.value = tc["txMac"]
+    eth1.dst.value = tc["rxMac"]
+
+    eth2.src.value = tc["innerTxMac"]
+    eth2.dst.value = tc["innerRxMac"]
+
+    ip.src.value = tc["txIp"]
```
Comment (ashutshkumr) [inline features/flows/headers/vxlan/test_vxlan_header_with_inner_ipv6_traffic.py:74]: Please add config and validation for tx/rx IPv6 address

Comment (Vibaswan) [reply features/flows/headers/vxlan/test_vxlan_header_with_inner_ipv6_traffic.py:76]: added config as well as validation for both ip versions

**Review by ashutshkumr — APPROVED** [review]

**Pull request merged**